src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -46,68 +48,74 @@ public function update() {
    *
    * @validateDrushConfig
    * @executeInVm
+   * @throws \Robo\Exception\TaskException
+   * @throws BltException
+   * @throws \Exception
    */
   public function import() {
     $strategy = $this->getConfigValue('cm.strategy');
 
-    if ($strategy != 'none') {
-      $cm_core_key = $this->getConfigValue('cm.core.key');
-      $this->logConfig($this->getConfigValue('cm'), 'cm');
-      $task = $this->taskDrush();
+    if ($strategy == 'none') {
+      // Still clear caches to regenerate frontend assets and such.
+      $result = $this->taskDrush()->drush("cache-rebuild")->run();
+      return $result;
+    }
 
-      $this->invokeHook('pre-config-import');
```

Comment:
Sorry the git diff is mangled from here on. All I did was to remove the rest of this function from the conditional.